sklearn/tests/test_common.py

```diff
@@ -324,4 +323,5 @@ def test_strict_mode_parametrize_with_checks(estimator, check):
 @pytest.mark.parametrize("estimator", N_FEATURES_IN_AFTER_FIT_ESTIMATORS,
                          ids=_get_check_estimator_ids)
 def test_check_n_features_in_after_fitting(estimator):
+    _set_checking_parameters(estimator)
```

Comment:
@thomasjpfan if you work on other module you might need this as long as this check is not part of the list of standard checks.